texthero/preprocessing.py

```diff
@@ -17,6 +17,44 @@
 
 from typing import List, Callable
 
+# REGEX pattern constants
```

Comment:
Should probably add a docstring like
```
"""
Define all the patterns for ...
"""
```

here to make life easier for developers later on.